static_report/src/sdlc/embed-window-data.js

```diff
@@ -27,6 +30,16 @@ const insertDataToHTML = async () => {
   log(PATH_TO_SINGLE_REPORT_DATA_JSON);
   log(PATH_TO_COMPARISON_REPORT_DATA_JSON);
 
+  // Set PipeRider Metadata
+  let metadata = {
```

Comment:
Suggestion: instead of exporting `PIPERIDER_VERSION`, and importing it here, 
create `getMetadata` function and export it from `core.js`, and opt for async non-blocking `readFile` over `readFileSync`.